crates/uv-interpreter/src/interpreter.rs

```diff
@@ -35,11 +35,7 @@ pub struct Interpreter {
 
 impl Interpreter {
     /// Detect the interpreter info for the given Python executable.
-    pub(crate) fn query(
-        executable: &Path,
-        platform: &Platform,
-        cache: &Cache,
-    ) -> Result<Self, Error> {
+    pub fn query(executable: &Path, platform: &Platform, cache: &Cache) -> Result<Self, Error> {
```

Comment:
Nit: It might be worth adding a comment here why the function is public. I could otherwise see myself to make it `pub(crate)` in the future when I don't see any public usages or we should have write up a document listing the APIs that are considered *stable*/  used downstream

Reply:
Sounds like a good idea, if I encounter it again, I'll make sure to include it in the comment :)